### PR TITLE
Make graphql endpoint port configurable

### PR DESCRIPTION
**Pull request by Bogoev91**
### Describe the background of your pull request

Endpoints through which client and server communicate need to be configurable

What does your pull request do? Does it solve a bug (which one?), add a feature?

It makes those endpoints/ports configurable through environment variables

### Additional context

Demo is not yet updated

### Governance 

- [x] Documentation is added
- [ ] Test cases are added or updated
- [x] I've read the contributing document https://github.com/cherrytwist/.github/blob/master/CONTRIBUTING.md
- [x] I've read and understand the Code of Conduct https://github.com/cherrytwist/.github/blob/master/CODE_OF_CONDUCT.md
- [x] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/cherrytwist/Coordination/blob/master/LICENSE 
     and the contributor license agreement: tba

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
   "license": "ISC",
   "scripts": {
     "start": "nodemon -w src --ext ts --exec ts-node src/server.ts",
+    "start-local":"nodemon -w src --ext ts --exec ts-node -r dotenv/config src/server.ts",
```
Comment (atstoyanov) [inline package.json:10]: What is the point of start-local? If won't be used for development nodemon is not required. 
Also "start:local" is more suitable name for the scripts. This allow having different purpose of same script. Like (dev, production etc.).

File: src/server.ts [hunk 2]
```diff
@@ -30,9 +30,9 @@ const main = async () => {
   const app = express();
   apolloServer.applyMiddleware({ app });
   app.listen(
```
Comment (atstoyanov) [inline src/server.ts:35]: With this solution we rely on having the endpoint set. There should be a default value or configuration validation:
- add default post ( const serverPort = process.env.GRAPHQL_ENDPOINT_PORT || 4000)
- validate configuration (check ConfigurationValidator)

**Review by atstojanov — DISMISSED** [review]
Include the new environment variable in the README.md

File: src/server.ts (hunk 2 above)
Comment (valentinyanakiev) [reply src/server.ts:35]: const ENDPOINT_PORT :)

File: package.json (hunk 1 above)
Comment (valentinyanakiev) [reply package.json:10]: I agree here - remove this and leave it as it is.If we have a default port (see below) configuration for port 4000, this change here won't be necessary

**Review by valentinyanakiev — CHANGES_REQUESTED** [review]
Some little tweaks required.

**Review by valentinyanakiev — APPROVED** [review]

**Review by valentinyanakiev — APPROVED** [review]
👍

**Review by valentinyanakiev — APPROVED** [review]
👍

**Pull request merged**